ui/src/components/UserList.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { Icon, List } from 'semantic-ui-react'
 import { Party } from '@daml/types';
-import { User } from '@daml2ts/create-daml-app/lib/create-daml-app-0.1.0/User';
+import { User } from '@daml-ts/create-daml-app-0.1.0/lib/User';
```

Comment:
The idea is to import this using
```suggestion
import daml from '@daml-ts/create-daml-app-0.1.0';
```
and then use `daml.User.User` instead of `User`. I'll take care of it.

Reply:
Actually, that change happened after we froze the 0.13.55 release. So, we need to do it once we're switching to the next SDK.